client/utils/account-fees.tsx

```diff
@@ -165,23 +165,15 @@ export const formatMethodFeesTooltip = (
 			{ hasFees( accountFees.fx ) ? (
 				<div>
 					<div>Foreign exchange fee</div>
```

Comment:
I do know that this is not the part of the original issue, but do we need to wrap this into the translation helper or function?

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/8596/commits/d57ff7ca092c18753cfef7d891bf601eab44f8ae

Reply:
Thank you for pointing out. Created https://github.com/Automattic/woocommerce-payments/issues/8620 and will address in separate PR.